app/2.0/docs/tools/polymer-json.md

```diff
@@ -138,29 +138,66 @@ the `builds` property. This is equivalent to passing different CLI flags to the
-    * `excludes`: Optional, `String[]`. Excludes the listed paths of files or folders from inlining.
-    * `inlineCss`: Optional, `Boolean`. Specifies whether to inline external CSS file contents into `<style>` tags.
-    * `inlineScripts`: Optional, `Boolean`. Specifies whether to inline external JavaScript file contents into `<script>` tags.
-    * `rewriteUrlsInTemplates`: Optional, `Boolean`. Specifies whether to rewrite element attributes inside of templates when inlining HTML.
-    * `sourcemaps`: Optional, `Boolean`. Specifies whether to create identity source maps for inline scripts.
-* `html`:
-  * `minify`: If `true`, minify all HTML.
-* `css`:
-  * `minify`: If `true`, minify all CSS.
-* `js`:
-  * `minify`: If `true`, minify all JS.
-  * `compile`: If `true`, use babel to compile all ES6 JS down to ES5.
+*   `name`: An optional name for your build. If multiple builds are defined, the `name` property 
+    is required.
+
+*   `preset`: An optional preset name that your build configuration can inherit from. 
+    See below for more information.
+
+*   `addServiceWorker`: If `true`, generate a service worker for your application.
+
+*   `addPushManifest`: If `true`, generate an [HTTP/2 Push Manifest](https://github.com/GoogleChrome/http2-push-manifest) for your application.
+
+*   `swPrecacheConfig`: An optional configuration file for the generated service worker.
+
+*   `insertPrefetchLinks`: If `true`, insert prefetch link elements into your fragments so that 
+    all dependencies are prefetched immediately.
+
+*   `bundle`: You may provide a boolean (`true` to bundle your application; default is `false`), 
+    or an object containing bundling configuration options.
+    
+    *   `excludes`: A list of paths of files and/or folders that should not be inlined. 
+      
+    *   `inlineCss`: Inline external CSS file contents into `<style>` tags.
+
+    *   `inlineScripts`: Inline external Javascript file contents into `<script>` tags.
+    
+    *   `rewriteUrlsInTemplates`: Rewrite element attributes inside templates when inlining html.
+
+    *   `sourcemaps`: Create identity source maps for inline scripts.
+
+    *   `stripComments`: Remove all comments except those tagged `@license`, or starting with
+        `<!--!` or `<!--#`.
+
+*   `html`: An object containing a configuration option for HTML:
+    
+    *   `minify`: If `true`, minify all HTML.
+
+*   `css`: An object containing a configuration option for CSS:
+
+    *   `minify`: If `true`, minify all CSS.
+
+*   `js`: An object containing configuration options for Javascript:
+
+    *   `minify`: If `true`, minify all JS.
+
+    *   `compile`: If `true`, use babel to compile all ES6 JS down to ES5.
+
+*   `browserCapabilities`: Capabilities required for a browser to consume this build. 
+    Values include `es2015` and `push`. [See the canonical list](https://github.com/Polymer/prpl-server-node/blob/master/src/capabilities.ts). 
```

Comment:
I think it would be far better to link to the README:

https://github.com/Polymer/prpl-server-node#capabilities

Also, there are currently only three values, `es2015`, `push`, and `serviceworker`, so perhaps just list all three and say, "For more information, see the `prpl-server-node` documentation."

Longer term, think about including these definitions in this file, the way we do the presets.